tensorflow-examples/src/main/java/org/tensorflow/model/examples/fashionmnist/VGGModel.java

```diff
@@ -0,0 +1,293 @@
+/*
+ *  Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *  =======================================================================
+ */
+package org.tensorflow.model.examples.fashionmnist;
+
+import org.tensorflow.Graph;
+import org.tensorflow.Operand;
+import org.tensorflow.Session;
+import org.tensorflow.Tensor;
+import org.tensorflow.framework.optimizers.Adam;
+import org.tensorflow.framework.optimizers.Optimizer;
+import org.tensorflow.model.examples.mnist.data.ImageBatch;
+import org.tensorflow.op.Ops;
+import org.tensorflow.op.core.*;
```

Comment:
I'd prefer no star imports in the example code too, @karllessard what do you think?